docs/markdown/Reference-manual.md

```diff
@@ -635,13 +635,15 @@ Defined tests can be run in a backend-agnostic way by calling `mesontest` inside
     ctarget vcs_tag(...)
 ```
 
-This command detects revision control commit information at build time and places it in the specified output file. This file is guaranteed to be up to date on every build. Keywords are similar to `custom_target` and all of them are mandatory.
+This command detects revision control commit information at build time and places it in the specified output file. This file is guaranteed to be up to date on every build. Keywords are similar to `custom_target`.
 
-- `input` file to modify (e.g. `version.c.in`)
-- `output` file to write the results to (e.g. `version.c`)
-- `fallback` version number to use when no revision control information is present, such as when building from a release tarball
+- `input` file to modify (e.g. `version.c.in`) (required)
+- `output` file to write the results to (e.g. `version.c`) (required)
+- `fallback` version number to use when no revision control information is present, such as when building from a release tarball (required)
+- `commmand` string list with the command to execute, see [`custom_target`](#custom_target) for details on how this command must look like
```

Comment:
tried that but it doesn't quite sound like correct english, reworded the sentence instead